docs/learn-faq.md

```diff
@@ -81,6 +81,88 @@ This script helps you quickly identify which validator is the lowest-staked and
 
 It is also possible to copy both `own stake` and `other stake` on the Staking page to a spreadsheet and then do a simple calculation.
 
+## Myths
+
+Below we present some myths about Polkadot that we've seen in the community.
+We're not absolutely sure what the origin of these myths are -- maybe they arise
+from people who have not taken the time to understand Polkadot. Regardless,
+we have collected a rebuttal for each of the common myths below. Feel free to
+link out to this page if you see one of these myths propagated on discussion platforms
+of the internet. 
+
+### Polkadot is an "Ethereum-killer"
+
+Some people position blockchains antagonistically against each other, claiming
+that all of the new blockchains are by default "Ethereum killers". The line of
+thinking is that there is a limited market share for blockchain platforms and
+therefore all projects are competing and vying for the lion's share. There may
+be some truth to this in the wide ecosystem of blockchain projects, but not all
+of them.
+
+Polkadot and Ethereum have significant differences in their technology, governance
+ideology, and community that instead forms a complimentary whole instead of a
+divided pie. Polkadot is a network for parachain applications, which may likely
+be domain-specific blockchains. Ethereum is instead a smart contract chain that
+focuses on maximizing the decentralization of its architecture. For more on the
+differences between the two chains see the more comprehensive [comparison article](learn-comparison-ethereum).
+
+In order to be an Ethereum killer, Polkadot would have to be actively working
+against the progression of the Ethereum technology and ecosystem, which is not
+true. Instead, Web3 Foundation, the holders of the treasury from the DOT sales, 
+promotes and funds technology that benefits the web3 world as a whole.
+
+### On-chain governance is flawed, it can never work
+
+Blockchain governance is a very young field, and no one knows what works and
+what doesn't -- yet. Polkadot's governance arises from a couple key observations,
+1) blockchains need some way to adapt and evolve and 2) the only measure of value
+that is intrinsic to a blockchain is its native coin and therefore the best
+measure of stakeholders for the system are the token holders. There are of
+course caveats to the above two observations and that is why Polkadot's governance
+is still in its experimental phases. Polkadot takes the approach that it will try
+out best methods and observe how they work and tweak them until they're optimal.
```

Comment:
Might be worth mentioning Kusama's "move fast and don't be afraid of change" attitude here as a perfect testing ground for novel formats of on-chain democracy, for the benefit of everyone's knowledge. 

Also I wouldn't say this addresses the myth per-se - I think we should maybe lead by listing some successful governance examples so far, passing referenda that had a big impact and would have caused hard forks otherwise, but also be honest and list some failures as well. Notably, long-term finality hold-ups are interesting and a problem I'm not sure Eth2 has solved either - it's easy to slash when nodes are offline or double sign, but online and out of sync created a slashless finalization stall which is I think an interesting problem to identify this early.